.github/workflows/build.yml

```diff
@@ -102,6 +102,8 @@ jobs:
             uarch-ram.bin
             uarch-pristine-ram.c
             uarch-pristine-hash.c
```

Comment:
Should we remove these `.c` and `.h` files now? Since the diff fulfills the purpose? Somewhat redundant.